cypress/fixtures/hosts.json

```diff
@@ -2389,7 +2389,347 @@
       "stale_warning_timestamp": "1958-08-14T23:00:00.0Z",
       "stale_timestamp": null,
       "mac_addresses": null,
-      "groups": []
+      "groups": {
+          "total": 37,
+          "count": 37,
+          "page": 1,
+          "per_page": 50,
+          "results": [
```

Comment:
`groups` is a list of groups that a particular host has. You must probably copy-pasted response from /groups which is not correct here.

Reply:
Removed the meta data from the groups, removed the results property.
Now the groups is just an array of objects